contrib/standalone-installer/make.py

```diff
@@ -85,8 +85,31 @@ def download_cab_file (directory, uri):
     subprocess.run (cmd, cwd=directory, check=True)
 
 def download_flatpak (directory):
-    cmd = ['wget', 'https://people.freedesktop.org/~hughsient/temp/fwupd.flatpak', '-O', 'fwupd.flatpak']
-    if 'DEBUG' in os.environ:
+    # check if installed yet
+    dep = 'org.freedesktop.fwupd'
+    cmd = ['flatpak', 'info', dep]
+    verbose = 'DEBUG' in os.environ
+    with open(os.devnull, 'w') as devnull:
+        if not verbose:
+            output = devnull
+        else:
+            output = None
+        if verbose:
+            print(cmd)
+        ret = subprocess.run (cmd, stdout=output, stderr=output)
+        #not installed
+        if ret.returncode != 0:
+            # install into local user's repo
+            cmd = ['flatpak', 'install', '--user',
+                'https://www.flathub.org/repo/appstream/org.freedesktop.fwupd.flatpakref', '--no-deps', '-y']
+            if verbose:
+                print(cmd)
+            subprocess.run (cmd, cwd=directory, check=True)
+
+    # generate a bundle
+    cmd = ['flatpak', 'build-bundle', os.path.join(os.getenv('HOME'),'.local', 'share', 'flatpak', 'repo'),
```

Comment:
I guess this is the best we can do, I don't think flathub builds bundles when doing builds...

Reply:
Yeah as far as I could tell they don't.  I'm somewhat torn one whether to do this build a bundle in this tool or just drop a fwupd flatpak ref.

On one hand you're going to need to have all the 3.30 platform deps so you'll need web access either way.
On the other hand you can "easily" install those and then this really does work offline.